src/main/java/org/veupathdb/service/eda/Resources.java

```diff
@@ -197,6 +203,14 @@ public static String getUserDbSchema(String projectId) {
     return SCHEMA_MAP.get(projectId);
   }
 
+  public static String getVdiDatasetsSchema() {
+    return "vdi_datasets_" + Environment.getRequiredVar("VDI_SCHEMA_SUFFIX");
```

Comment:
May be better to put this in a static field so getRequiredVar() gets called right away.  I could see the schema getter not being called until we need to execute some SQL.

Reply:
Thanks for the quick response! I'll fix this in the follow-up PR to start using this schema. I'm going to merge this to fix up dev.